src/main/java/dev/slimevr/vr/processor/TransformNode.java

```diff
@@ -78,4 +78,15 @@ public void combineWithParentGlobalRotation(Transform parent) {
 			.multLocal(worldTransform.getTranslation())
 			.addLocal(parent.getTranslation());
 	}
+
+	public void detachCompletely() {
```

Comment:
Please rename to something like `detachWithChildren()`

Reply:
how about `detachParentAndChildren()`?

Reply:
It doesn't detaches parent, it detaches from parent though